evap/results/tools.py

```diff
@@ -10,6 +10,8 @@
 from evap.evaluation.models import TextAnswer, Contribution, RatingAnswerCounter
 from evap.evaluation.tools import questionnaires_and_contributions
 
+import itertools
+
```

Comment:
move that to the other stdlib import above please

Reply:
check. i'm really looking forward to deciding on an explicit order and using a linter for that :)

Reply:
i know! consistent import order is one of the few things i really liked better in 1327 so far.